src/site/content/en/blog/promises/index.md

```diff
@@ -559,7 +559,7 @@ above as a flowchart (because I love flowcharts):
 
 <div style="position: relative; padding-top: 93%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;overflow:hidden"
-   src="/promises/promise-flow.svg" frameborder="0" allowtransparency="true"></iframe>
+   src="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/simQvoUExWisIW0XxToH.svg" | imgix }}" frameborder="0" allowtransparency="true"></iframe>
```

Comment:
Double check this works the same.

Also was wondering if we should use the iframe shortcode here...

Reply:
It works the same, though the iframe lets it easily scale. I'm not sure how we could correct that considering we use height/widths

Reply:
ah yeah then let's leave it as is